src/frontend/src/components/SearchBar/SearchBar.jsx

```diff
@@ -138,14 +155,7 @@ function CustomizedInputBase(props) {
           </Grid>
           <Grid item xs={12} sm={10} lg={10}>
             <FormControl fullWidth={true}>
-              <TextField
-                className={classes.input}
-                placeholder="How to Get Started in Open Source"
-                inputProps={{ 'aria-label': 'search telescope' }}
-                variant="outlined"
-                value={text}
-                onChange={(event) => onTextChange(event.target.value)}
-              />
+              <SearchInput filter={filter} text={text} onTextChange={onTextChange} />
```

Comment:
Let's rename `onTextChange` to `onChange` to be more consistent with the DOM naming.

Reply:
Done.